src/runtime/components/Tabs.vue

```diff
@@ -51,7 +52,8 @@ import { reactivePick } from '@vueuse/core'
 const props = withDefaults(defineProps<TabsProps<T>>(), {
   content: true,
   defaultValue: '0',
-  orientation: 'horizontal'
+  orientation: 'horizontal',
+  size: 'md'
```

Comment:
The default `size` shouldn't be in the component but in the theme `defaultVariants`: https://github.com/benjamincanac/ui3/pull/121/files#diff-af2cb38fdb444c8e0856e185704aa0ecd42e5b0f07cf8cb8c9fe135bd9fd9412R133